internal/attestation/measurements/measurements.go

```diff
@@ -38,6 +38,13 @@ const (
 // M are Platform Configuration Register (PCR) values that make up the Measurements.
 type M map[uint32]Measurement
 
+// WithMetadata is a struct supposed to provide CSP & image metadata next to measurements.
+type WithMetadata struct {
```

Comment:
This data type currently has no use in our main code.
Do we expect to use it in the future?
I could see adapting the `measurements.M` unmarshal methods to either take a file with metadata or without, in which case we probably don't have to export this type.

Reply:
`fetch-measurements` would likely require it in some way.